ow_system_plugins/admin/controllers/permissions.php

```diff
@@ -72,7 +73,16 @@ public function index()
         $guestsCanView->setDescription($language->text('admin', 'permissions_idex_if_not_yes_will_override_settings'));
         $form->addElement($guestsCanView);
 
-        $password = new PasswordField('password');
+        $password = new TextField('password');
+        $password->setHasInvitation(true);
+        if($baseConfigs['guests_can_view'] == 3)
+        {
+            $password->setInvitation("Change password");
+        }
+        else
+        {
+            $password->setInvitation("Add password");

```

Comment:
You should use Core Localisation system. I mean: $password->setInvitation(OW::getLanguages()->text('admin', 'change_password')); instead of $password->setInvitation("Add password");